src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineView.xaml.cs

```diff
@@ -72,11 +73,40 @@ public DocumentOutlineView(
         _windowSearchHost = windowSearchHostFactory.CreateWindowSearchHost(SearchHost);
         _windowSearchHost.SetupSearch(this);
 
+        this.PreviewKeyDown += OnPreviewKeyDown;
         viewTracker.CaretMovedOrActiveViewChanged += ViewTracker_CaretMovedOrActiveViewChanged;
     }
 
+    private void OnPreviewKeyDown(object sender, KeyEventArgs e)
+    {
+        _threadingContext.ThrowIfNotOnUIThread();
+
+        // If focus is in the Commands toolbar, move it to the Search box on Down or Tab
+        if ((e.Key == Key.Down || e.Key == Key.Tab) && Keyboard.Modifiers == ModifierKeys.None && Commands.IsKeyboardFocusWithin)
+        {
+            if (_windowSearchHost is not null)
+            {
+                _windowSearchHost.Activate();
+                e.Handled = true;
+            }
+        }
+        // If focus is in the Search box, move it to the SymbolTree on Down or Tab
+        else if ((e.Key == Key.Down || e.Key == Key.Tab) && Keyboard.Modifiers == ModifierKeys.None && SearchHost.IsKeyboardFocusWithin)
```

Comment:
Do we need to be supporting shift/tab to go back in the other direction?

Reply:
I tried shift-tab and that works, it was only the tabbing into/arrowing down into the search that was broken. There was also no tab/down from the search to the tree